javascript/new-src/src/factories/ola.js

```diff
@@ -22,6 +22,19 @@
 ola.factory('$ola', ['$http', '$window', 'OLA',
   function($http, $window, OLA) {
     'use strict';
+    var highestChannelNumberUsed = 0;
```

Comment:
Does this need resetting when you switch from one universe to another?

Reply:
Oh, I didn't check this. But I guess some kind of reset is needed.

Reply:
changing the universe resets `highestChannelNumberUsed` now.

Reply:
Great thanks.